code/web/sys/LibraryLocation/Library.php

```diff
@@ -59,7 +59,6 @@ class Library extends DataObject {
 	public $headerText;
 	public $footerText;
 	public $systemMessage;
-
 	public $generateSitemap;
```

Comment:
May be worth reverting this line removal.. Whilst I agree, it cleans the file up a little.. I'm not sure if it's worth touching this file as part of this overall patch set.

Rather nitpicky.. Apologies.